server/index.js

```diff
@@ -0,0 +1,38 @@
+// connect express
+const express = require('express');
+
+const app = express();
+
+const path = require('path');
+
+const port = 7777;
+
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
+
+// serve client files
+app.use(express.static(path.join(__dirname, '/../client')));
+
+// import DB
+const db = require('../database');
+
+// GET request
+app.get('/booking', (req, res) => {
+  db.find((data) => {
```

Comment:
what is the data here in the parameter supposed to be? How are you getting it?

Reply:
@jennqiao This GET request is supposed to fetch all data in DB. 'data' parameter is used by callback to access data in DB.